src/Illuminate/Support/Arr.php

```diff
@@ -100,6 +100,23 @@ public static function dot($array, $prepend = '')
     }
 
     /**
+     * Expand dot-flattened array into a multi-dimensional associative array
```

Comment:
No, you've missed the full stop. All sentences must end in a full stop in English...